### PR TITLE
Fix #4884 - enable Gens balsamic button

### LINKED ISSUE #4884: GENS button for TGA samples
**Is your feature request related to a problem in the current program to new available technology or software? Please describe and add links/citations if appropriate.**

In Scout for production cases of the WGS sequencing type the CN profile button exists which takes the user to view the CN profile in GENS. This button is missing for TGA as the CN profile was not uploaded previously, however with version 16 of balsamic this is planned to be implemented.

CG PR: https://github.com/Clinical-Genomics/cg/pull/3361
Balsamic PR: https://github.com/Clinical-Genomics/BALSAMIC/pull/1465

Tested upload in stage, and saw that the link from a CNV variant works as expected! 

**Describe the solution you'd like**

Make the CN profile button appear for TGA cases

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Gens button enabled also on cancer individuals table (if gens integration is enabled).

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN

### CODE REVIEW

**Comment by dnil** [comment]
❌  Before: Screenshot 2024-09-23 at 14 15 23
✅ After: Screenshot 2024-09-23 at 14 19 17

**Comment by dnil** [comment]
Ping @mathiasbio - its on stage now, enabled for all `cancer` track cases as well. Is that as intended?

File: scout/server/blueprints/cases/templates/cases/individuals_table.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
         <tbody>
           {% for ind in case.individuals %}
             <tr {% if ind.phenotype_human == 'tumor' %} class="bg-danger-light" {% endif %}>
-              <td>{{ ind.display_name }}{% if gens_info.display and ind.analysis_type|upper == "WGS" %}
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/individuals_table.html:32]: 👍🏻

**Review by northwestwitch — COMMENTED** [review]
Nice fix! 💯

**Review by northwestwitch — APPROVED** [review]

**Comment by mathiasbio** [comment]
Sorry for the late reply! Looks like it's working 🙏 Thank you! I don't know how this implementation works, if it will display the button even if GENS results are not uploaded. But either way it should be fine. We're planning to make this release SOON! (I keep saying this but it is true) and if the button is showing and not leading anywhere in the interim time it should be fine!

**Comment by dnil** [comment]
Good, and quite agree. It will be nice when they are loaded, but reaching a "missing gens data" page in the meantime is no biggie.

There is an old issue on gens to allow api polling for case data. Feel free to work on/nudge that one if you would wish to have the buttons more adaptive.

**Pull request merged**